### PR TITLE
Added README for concurrent ruby 

### PR DESCRIPTION
**Pull request by eunice98k**
This PR adds documentation for Concurrent Ruby, closing [#234.](https://github.com/open-telemetry/opentelemetry-ruby/issues/234)

I have added a README document describing the Concurrent ruby gem, installation methods and its license.

### CODE REVIEW

File: instrumentation/concurrent_ruby/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# OpenTelemetry Concurrent Ruby Instrumentation
+
+The OpenTelemetry Concurrent Ruby gem is a community maintained instrumentation for the [Concurrent Ruby][concurentruby-home]. This is a concurrency toolkit inspired from various languages and classic concurrency patterns.
```
Comment (ericmustin) [inline instrumentation/concurrent_ruby/README.md:3]:
```suggestion
The OpenTelemetry Concurrent Ruby gem is a community maintained instrumentation for the [Concurrent Ruby][concurrent-ruby-home]. This is a concurrency toolkit inspired from various languages and classic concurrency patterns. 
```

File: instrumentation/concurrent_ruby/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+The `opentelemetry-instrumentation-concurrent_ruby` gem source is [on github][repo-github], along with related gems including `opentelemetry-api` and `opentelemetry-sdk`.
+
+The OpenTelemetry Ruby gems are maintained by the OpenTelemetry-Ruby special interest group (SIG). You can get involved by joining us on our [gitter channel][ruby-gitter] or attending our weekly meeting. See the [meeting calendar][community-meetings] for dates and times. For more information on this and other language SIGs, see the OpenTelemetry [community page][ruby-sig].
+
+## License
+
+Apache 2.0 license. See [LICENSE][license-github] for more information.
+
+[concurentruby-home]: https://github.com/ruby-concurrency/concurrent-ruby
```
Comment (ericmustin) [inline instrumentation/concurrent_ruby/README.md:43]:
```suggestion
[concurrent-ruby-home]: https://github.com/ruby-concurrency/concurrent-ruby
```

**Review by ericmustin — CHANGES_REQUESTED** [review]
2 extremely minor changes. otherwise this LGTM, nice work, thanks @eunice98k !

**Review by ericmustin — APPROVED** [review]
👍

**Review by alolita — APPROVED** [review]
ltgm

**Review by fbogsany — APPROVED** [review]

**Pull request merged**